src/timings/Timings.php

```diff
@@ -338,7 +338,7 @@ public static function getAsyncTasksErrorTimings(AsyncTask $task, string $group
 		if(!isset(self::$asyncTasksError[$taskClass])){
 			self::$asyncTasksError[$taskClass] = new TimingsHandler(
 				"Error - " . self::shortenCoreClassName($taskClass, "pocketmine\\"),
-				self::$schedulerAsync,
+				self::$schedulerAsync ?? null,
```

Comment:
It would be better to use `Timings::init()` to ensure the proper structure is maintained.

Reply:
This means that in the case of unit testing, we're forming completely useless handlers. Wouldn't it just be better to have a function that retrieves the handler from the async task, if it's not defined (init not called) that creates it? Like : 
```php
public static function getSchedulerAsyncTiming() : TimingsHandler;
```

Reply:
After all, this could also happen to other handlers in the future, so why not generalize the init of Timings even during unit tests, rather than making a specific case for AsyncTask?

Reply:
> This means that in the case of unit testing, we're forming completely useless handlers. Wouldn't it just be better to have a function that retrieves the handler from the async task, if it's not defined (init not called) that creates it? Like :
> 
> ```
> public static function getSchedulerAsyncTiming() : TimingsHandler;
> ```

This adds extra opcodes and a function call, increasing the performance influence of timings. Timings should minimally influence performance.

You can simply call Timings::init() within these functions to ensure they are available. We may want to unit test timings themselves in the future also, so hierarchy should be preserved.